.gitignore

```diff
@@ -4,3 +4,4 @@ content/**.gen.*
 *.txt
 cmd
 server.go
+.history
```

Comment:
Why need this?

Reply:
It's generated by VS code, including the change history of code.

Reply:
Please revert this change.